equinix/fabric_mapping_helper.go

```diff
@@ -0,0 +1,706 @@
+package equinix
+
+import (
+	"fmt"
+	v4 "github.com/equinix-labs/fabric-go/fabric/v4"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"log"
+	"math/rand"
+	"time"
+)
+
+func serviceTokenToFabric(serviceTokenRequest []interface{}) v4.ServiceToken {
+	mappedST := v4.ServiceToken{}
+	for _, str := range serviceTokenRequest {
+		stMap := str.(map[string]interface{})
+		stType := stMap["type"].(interface{}).(string)
+		uuid := stMap["uuid"].(interface{}).(string)
+		stTypeObj := v4.ServiceTokenType(stType)
+		mappedST = v4.ServiceToken{Type_: &stTypeObj, Uuid: uuid}
+	}
+	return mappedST
+}
+
+func additionalInfoToFabric(additionalInfoRequest []interface{}) []v4.ConnectionSideAdditionalInfo {
+
+	var mappedaiArray []v4.ConnectionSideAdditionalInfo
+	for _, ai := range additionalInfoRequest {
```

Comment:
```suggestion
	for i, ai := range additionalInfoRequest {
```

and then you can remove `i := 0` and `i++`.

Reply:
Good point, not sure how I missed it. Will take care.